### PR TITLE
Add instances information and shrink options to resource_vkcs_db_cluster

### PR DESCRIPTION
**Pull request by schirevko**

### CODE REVIEW

File: vkcs/db_cluster.go [hunk 1]
```diff
@@ -24,6 +24,69 @@ func flattenDatabaseClusterShard(inst dbClusterInstanceResp) map[string]interfac
...
+					needToRemain = true
+				}
+			}
+			if !needToRemain {
+				ids = append(ids, dbClusterShrinkOpts{ID: instance.ID})
+				foundIDs++
+			}
+		}
+		if foundIDs != toDelete {
+			return nil, fmt.Errorf("invalid shrink options")
```
Comment (ftersin) [inline vkcs/db_cluster.go:83]: explain more details in the message, e.g. "eigther wrong instance id, or wrong instance number specified" etc

Comment (schirevko) [reply vkcs/db_cluster.go]: expanded error messages

File: vkcs/db_cluster.go [hunk 2]
```diff
@@ -24,6 +24,69 @@ func flattenDatabaseClusterShard(inst dbClusterInstanceResp) map[string]interfac
...
+				if instance.ID == opt {
+					needToRemain = true
+				}
+			}
+			if !needToRemain {
+				ids = append(ids, dbClusterShrinkOpts{ID: instance.ID})
+				foundIDs++
+			}
+		}
+		if foundIDs != toDelete {
```
[the remark this answers is not in the document]
Comment (schirevko) [reply vkcs/db_cluster.go:86]: added shrinkOptions validation

File: vkcs/resource_vkcs_db_cluster.go [hunk 3]
```diff
@@ -26,6 +26,10 @@ var (
 	dbClusterStatusBackup             dbClusterStatus = "BACKUP"
 )
 
+var (
+	dbClusterInstanceRoleLeader string = "leader"
```
Comment (pashapdev) [inline vkcs/resource_vkcs_db_cluster.go:30]: можно константой сделать? нигде не вижу вроде &dbClusterInstanceRoleLeader

**Review by pashapdev — APPROVED** [review]

File: vkcs/resource_vkcs_db_cluster.go (hunk 3 above)
Comment (schirevko) [reply vkcs/resource_vkcs_db_cluster.go:30]: сделал константой

**Pull request merged**